universal-application-tool-0.0.1/app/services/program/ProgramDefinition.java

```diff
@@ -50,6 +50,9 @@ public static Builder builder() {
   /** An external link to a page containing more details for this program. */
   public abstract String externalLink();
 
+  /** Whether the program should be hidden from the Applicant's program overview page. */
+  public abstract boolean hideFromView();
```

Comment:
suggestion (blocking): this could would be a bit more open to extension if instead of using a boolean `hideFromView` field it used an enum `displayMode` value. For now it would just have two possible values: `PUBLIC` and `HIDDEN_IN_INDEX`, but could be extended with other options they wanted for example to have another mode that also changes the behavior of the deep link.

Reply:
Done